db/migrate/20170903041257_ids_to_bigints.rb

```diff
@@ -0,0 +1,18 @@
+class IdsToBigints < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
It would be better if it is reversible (currently it is not because of the regex.)

Reply:
Given that we currently have a mix of `bigint` and `integer` types, the only way to do this is to list them all explicitly. If that's desirable, I can do it. (Mostly, I did it this way so I could move the migration around to catch the latest columns before this gets merged. Now that it's actually getting merged, explicitly naming all the `integer` _id columns is fine, if lengthy.)